Gulpfile.js

```diff
@@ -372,9 +372,6 @@ const lint = eslint(".");
 lint.displayName = "lint";
 task("lint", lint);
 task("lint").description = "Runs eslint on the compiler and scripts sources.";
-task("lint").flags = {
-    "   --ci": "Runs eslint additional rules",

```

Comment:
For posterity, there are no additional rules; all rules already run. Probably this is a leftover from tslint or something.